clever/src/selfcheck.py

```diff
@@ -731,6 +766,7 @@ def selfcheck():
     check_optical_flow()
     check_vpe()
     check_rangefinder()
+    check_hw_health()
```

Comment:
Maybe move it higher? After clever service.

Reply:
My idea was to put it right over the CPU load check, so that the user would immediately have an idea of what's causing the CPU to be throttled.

Reply:
OK.